platoon/mpi_util.py

```diff
@@ -34,9 +34,9 @@ def launch_mpi_workers(workers_count, experiment_name, worker_args):
     args = shape_args(experiment_name, worker_args, "worker")
     info = MPI.Info.Create()
     info['host'] = socket.gethostname()
-    #  info['ompi_non_mpi'] = 'true'
     env = dict(os.environ)
-    info['env'] = '\n'.join(env.keys())
+    keep_list = ['THEANO_FLAGS', 'PLATOON_TEST_WORKERS_NUM', '_']
+    info.Set(key='env', value='\n'.join('%s=%s' % (key, env[key]) for key in keep_list if key in env))
```

Comment:
If this ultimately works (i.e. worker processes have the same environmental variables as controllers) without having to set `info['env']`, then let's please omit it so that we may not need to specify extra variables like `PLATOON_TEST_WORKERS_NUM` or `_`.